### PR TITLE
Validation: Duplicate alt text and missing description error fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
## Summary

Fixed:

- description-missing errors (description metadata missing in articles)
- image-alt-text-duplicated errors (duplicate alt text descriptions within an article)

This PR fixes 31 issues across 23 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dotnet-desktop-guide/framework/winforms/advanced/power-management-in-windows-forms.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Power Management"
+description: Learn about how Windows Forms applications can use the power management features in the Windows operating system to monitor a computer's power status and take action.
```
Comment (JustPies) [inline dotnet-desktop-guide/framework/winforms/advanced/power-management-in-windows-forms.md:3]: This is over the 160 character max.

**Comment by brittmsantos** [comment]
**Global effort to fix validation errors**

@adegeo

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/winforms/advanced/restricting-the-drawing-surface-in-gdi.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Restricting the Drawing Surface in GDI+"
+description: Learn how to restrict the drawing surface in GDI+ using regions, which can be constructed from paths and contain the outlines of strings.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/restricting-the-drawing-surface-in-gdi.md:3]:
```suggestion
description: Learn how to restrict the drawing surface in GDI+ using clipping regions in Windows Forms.
```

File: dotnet-desktop-guide/framework/winforms/advanced/polygons-in-gdi.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Polygons in GDI+"
+description: Learn about how to draw a polygon in GDI+ using a System.Drawing.Graphics object, a System.Drawing.Pen object, and an array of System.Drawing.Point objects.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/polygons-in-gdi.md:3]:
```suggestion
description: Learn about how to draw a polygon in GDI+ using a Graphics object, a Pen object, and an array of objects.
```

File: dotnet-desktop-guide/framework/winforms/advanced/overview-of-graphics.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Overview of Graphics"
+description: Learn how the GDI+ application programming interface (API) forms the subsystem of the Microsoft Windows OS and displays information on screens and printers.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/overview-of-graphics.md:3]:
```suggestion
description: Learn how the GDI+ application programming interface (API) forms the subsystem of the Microsoft Windows operating system, which displays information on screens and printers.
```

File: dotnet-desktop-guide/framework/winforms/advanced/open-and-closed-curves-in-gdi.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Open and Closed Curves in GDI+"
+description: Learn more about how GDI+ handles open and closed curves with the System.Drawing.Graphics class and its methods.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/open-and-closed-curves-in-gdi.md:3]:
```suggestion
description: Learn more about how GDI+ can create open and closed curves with the Graphics class and its methods.
```

File: dotnet-desktop-guide/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: International fonts in forms and controls
+description: Learn about using font fallback as the recommended method for selecting International fonts in Windows Forms and controls.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/international-fonts-in-windows-forms-and-controls.md:3]:
```suggestion
description: Learn about using font fallback as the recommended method for selecting international fonts in Windows Forms and controls.
```

**Pull request merged**